### ISSUE TITLE
Uncaught TypeError: Cannot read property 'message' of null

### ISSUE THREAD

**Issue by bhanuchnd** [TypeStrong/atom-typescript#1423]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.27.0 x64
**Electron**: 1.7.11
**OS**: Mac OS X 10.13.4
**Thrown From**: [atom-typescript](https://github.com/TypeStrong/atom-typescript) package 12.4.1

### Stack Trace

Uncaught TypeError: Cannot read property 'message' of null

```
At /Users/krishnavamsi/.atom/packages/atom-typescript/dist/client/client.js:84

TypeError: Cannot read property 'message' of null
    at Immediate.setImmediate (/packages/atom-typescript/dist/client/client.js:84:37)
    at runCallback (timers.js:672:20)
    at tryOnImmediate (timers.js:645:5)
    at processImmediate [as _immediateCallback] (timers.js:617:5)
```

### Commands

```
```

### Non-Core Packages

```
atom-clock 0.1.16 
atom-typescript 12.4.1 
busy-signal 1.4.3 
hyperclick 0.1.5 
intentions 1.1.5 
linter 2.2.0 
linter-ui-default 1.7.1 
platformio-ide-terminal 2.8.1 
```

**Comment by lierdakil** [comment]
What TypeScript version are you using?

**Comment by lierdakil** [comment]
Nevermind, I'm pretty sure I figured it out. Should be fixed in v12.5.0. Thanks for reporting!
